caldav/elements.go

```diff
@@ -21,8 +21,16 @@ var (
 	calendarQueryName    = xml.Name{namespace, "calendar-query"}
 	calendarMultigetName = xml.Name{namespace, "calendar-multiget"}
 
-	calendarName     = xml.Name{namespace, "calendar"}
-	calendarDataName = xml.Name{namespace, "calendar-data"}
+	calendarName      = xml.Name{namespace, "calendar"}
+	calendarDataName  = xml.Name{namespace, "calendar-data"}
+	calendarColorName = xml.Name{
+		Space: "http://apple.com/ns/ical/",
```

Comment:
The standard way to set the color of a calendar is: https://datatracker.ietf.org/doc/html/rfc7986#section-5.9

This is a non-standard, Apple-specfic property. I'm not sure I want to support it.

Reply:
I never saw this one in real life. But happy to add this as well.

Reply:
Just adding my 2 cents: I have indeed also observed the proliferation of  this pseudo-standard (apple.com/ns/ical) into many open-source clients (e.g. DavX5 sets it on MKCOL). On the other hand, from my experience, after initial calendar creation, this is treated as a pure client-side property by pretty much all clients (i.e. it never gets updated again), so not sure how necessary it really is...

Reply:
> so not sure how necessary it really is...

agreed - this is for sure a pretty low feature.
Never the less if the color is stored server side it will be "synced" once opening the calendar with another client which has some benefits of recognizing calendars.

e.g. on the owncloud 10 codebase (php+sabredav) colors are the same on all clients (owncloud calendar app in web, DAVx and iOS)

Reply:
Hm, actually it doesn't seem like there is any standard way to fetch a `VCALENDAR` for the calendar itself. So RFC 7986 is only useful to set the calendar color when exporting a whole calendar as .ics.

Oh well, let's just do like everybody else then…